app/src/main/java/com/janewaitara/businesscardapp/MainActivity.kt

```diff
@@ -2,11 +2,180 @@ package com.janewaitara.businesscardapp
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
 
+    /**
+     * Arrays of the business card properties from the string arrays in the Res folder in strings.xml
```

Comment:
You have done an excellent job commenting your code throughout and keeping it very readable and organized.